perception/autoware_image_projection_based_fusion/launch/segmentation_pointcloud_fusion.launch.py

```diff
@@ -0,0 +1,166 @@
+# Copyright 2025 Tier IV, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import launch
+from launch.actions import DeclareLaunchArgument
+from launch.actions import OpaqueFunction
+from launch.actions import SetLaunchConfiguration
+from launch.conditions import IfCondition
+from launch.conditions import UnlessCondition
+from launch.substitutions import LaunchConfiguration
+from launch_ros.actions import Node
+from launch_ros.substitutions import FindPackageShare
+import yaml
+
+
+class SegmentationPointcloudFusion:
+    def __init__(self, context):
+        self.context = context
+        with open(LaunchConfiguration("sync_param_path").perform(context), "r") as f:
+            self.segmentation_pointcloud_fusion_sync_param = yaml.safe_load(f)["/**"][
+                "ros__parameters"
+            ]
+
+        with open(
+            LaunchConfiguration("segmentation_pointcloud_fusion_param_path").perform(context), "r"
+        ) as f:
+            self.segmentation_pointcloud_fusion_param = yaml.safe_load(f)["/**"]["ros__parameters"]
+
+        self.camera_ids = LaunchConfiguration("fusion_camera_ids").perform(context)
+        # convert string to list
+        self.camera_ids = yaml.load(self.camera_ids, Loader=yaml.FullLoader)
+        self.segmentation_pointcloud_fusion_param["rois_number"] = len(self.camera_ids)
+        mask_timestamp_offsets = []
+        approximate_camera_projection = []
+        rois_timestamp_noise_window = []
+        approximate_camera_projection = []
```

Comment:
remove this line

Reply:
thank you. I removed at https://github.com/autowarefoundation/autoware_universe/pull/10609/commits/7ee952dab1896c37a3283bf61a613059d9d1d23a